packages/core/src/dialects/abstract/query-interface.js

```diff
@@ -482,16 +480,10 @@ export class AbstractQueryInterface extends AbstractQueryInterfaceTypeScript {
 
     const modelDefinition = instance?.modelDefinition;
 
-    options = { ...options, model: instance?.constructor };
+    options = { ...options, where, model: instance?.constructor };
```

Comment:
model should not be passed via the option bag. If it's available, use it as the first parameter instead of `tableName`

Reply:
Updated now.